README.md

```diff
@@ -1,6 +1,14 @@
+# React Forms Lab
+
 [![Pursuit Logo](https://avatars1.githubusercontent.com/u/5825944?s=200&v=4)](https://pursuit.org)
 
-# [Assignment Title]
+## Functionality
+
+Build a React app containing:
+
+- An input where the user can enter the values in an array
+- A select to choose between calculating the sum, the average, and the [mode](https://www.mathsisfun.com/definitions/mode.html) (most frequently occurring number)
+- A calculate button
 
 ## Getting Started
```

Comment:
There are no jest tests so can you delete the line below here about `npm test to run jest / node tests`?

Reply:
Also can you delete `App.test.js`? Unless you think there's value in keeping it in the repo?